<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>509</width>
      <height>399</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>500</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>Modelos de lista</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonListaSeq">
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>Lista sequencial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonLSE">
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>Lista simplesmente encadeada</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonLDE">
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>Lista duplamente encadeada</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
